zsl/utils/injection_helper.py

```diff
@@ -68,7 +68,7 @@ def method_or_class_wrapper(*a, **kwargs):
             # Call the method.
             return inject_f(f)(*a, **kwargs)
 
-        if inspect.getargspec(f)[0][0] == 'self':
+        if inspect.ismethod(f):
```

Comment:
`getargspec` is not supported in Py3 and I think `ismethod` is the right method here anyway.